manage/sawtooth_manage/docker.py

```diff
@@ -154,20 +157,23 @@ def start(self, node_config):
         yaml.dump(compose_dict,
                   open(os.path.join(compose_dir, 'docker-compose.yaml'),
                        mode='w'))
+
+        print("Starting: {}".format(node_name))
```

Comment:
Printing within library code is usually not desirable, as you can then not use the library without it producing output.  A couple options: use logging instead, do it in a callback method.